src/main/java/git/Storage.java

```diff
@@ -62,28 +87,63 @@ private Grocery parseGrocery(String line) {
         Location location = parts[6].equalsIgnoreCase("null") ? null : new Location(parts[6].trim());
         return new Grocery(name, amount, threshold, expiration, category, cost, location);
-        //return new Grocery(name, 0, 0, null, "", 0, null);
     }
-
     /**
-     * Saves the current list of groceries to the file.
+     * Saves the current list of recipes to the file.
      *
-     * @param groceries The list of groceries to save.
+     * @param recipeArr The list of recipes to save.
      */
-    public static void saveFile(List<Grocery> groceries) {
+    public void saveRecipeFile(ArrayList<Recipe> recipeArr) {
         try {
             File directory = new File("./data");
             if (!directory.exists()) {
                 directory.mkdirs();
             }
-            FileWriter writer = new FileWriter("./data/groceryList.txt");
-            for (Grocery grocery : groceries) {
-                writer.write(grocery.toSaveFormat() + "\n");
+            FileWriter writer = new FileWriter("./data/recipeList.txt");
+            for (Recipe recipe : recipeArr) {
+                writer.write(recipe.toRecipeSaveFormat() + "\n");
             }
             writer.close();
         } catch (IOException e) {
-            System.out.println("An error occurred while saving groceries.");
+            System.out.println("An error occurred while saving recipes.");
             e.printStackTrace();
         }
     }
+    /**
+     * Loads recipes from the file.
+     * @return recipeList loaded from the file. If file does not exist, returns an empty recipeList.
+     */
+    public RecipeList loadRecipeFile(){
+        RecipeList recipeList = new RecipeList();
+        try {
+            File file = new File("./data/recipeList.txt");
+            Scanner scanner = new Scanner(file);
+            while (scanner.hasNextLine()) {
+                String line = scanner.nextLine();
+                Recipe recipe = parseRecipe(line);
+                recipeList.addRecipe(recipe);
+            }
+            scanner.close();
+        }
+        catch (FileNotFoundException e) {
+            //System.out.println("No saved recipes found.\n ");
+        }
+        return recipeList;
+    }
+    /**
+     * Parses a string from the file into a Recipe object.
+     *
+     * @param line The string to parse.
+     * @return The parsed recipe object.
+     */
+    private Recipe parseRecipe(String line) {
+        String[] parts = line.split(" \\| ");
+        String title = parts[0].trim();
+        String[] ingredientsArray = parts[1].equalsIgnoreCase("null") ? null : parts[1].split(", ");
+        ArrayList<String> ingredientsList = new ArrayList<>(Arrays.asList(ingredientsArray));
+        String[] stepsArray = parts[2].equalsIgnoreCase("null") ? null : parts[2].split(", ");
+        ArrayList<String> stepsList = new ArrayList<>(Arrays.asList(stepsArray));
+
```

Comment:
The transition to more specific file handling methods (loadGroceryFile, saveGroceryFile, loadRecipeFile, saveRecipeFile) is commendable for its clarity and future-proofing, as it allows further expansion, like adding more data types to handle.
Catching FileNotFoundException in loadRecipeFile without printing a message may silently fail. Consider logging this event or handling it in a way that informs the user or system about the missing file, improving fault tolerance.